tools/jslint.js

```diff
@@ -240,6 +246,11 @@ if (cluster.isMaster) {
     if (files instanceof Array) {
       // Lint some files
       const report = cli.executeOnFiles(files);
+
+      // If we were asked to fix the fixable issues, do so.
+      if (cliOptions.fix)
+        CLIEngine.outputFixes(report);
```

Comment:
Ok, I was just surprised it has enough information in the `report` to fix errors without needing the actual `cli` instance.